deploy/crds/operator.ibm.com_v1alpha1_operandconfig_cr.yaml

```diff
@@ -10,7 +10,7 @@ spec:
       meteringUI: {}
   - name: ibm-licensing-operator
     spec:
-      IBMLicensing: {}
+      iBMLicensing: {}
```

Comment:
@horis233 why change the name, if these names changed, we also need to update cicd test guide, https://github.ibm.com/IBMPrivateCloud/roadmap/blob/master/feature-specs/cicd/common-service-1Q-e2e-testing.md

Reply:
@DanielXLee Because when deleting a custom resource from the OperandConfig, ODLM can get custom name from the `kind` in the `alm-example`.  When updating status, I set the logic to delete the custom resource with the name as `kind` only lower its first letter. If we don't set this naming rule, we need to use a loop to check the CRs in the operator. 

If you think it is feasible, I will update the e2e testing in the roadmap afterwards.